cmd/server/main.go

```diff
@@ -96,6 +97,7 @@ func main() {
 	disclaimerRepo := documents.NewDisclaimerRepo(logger, db)
 	documentRepo := documents.NewDocumentRepo(logger, db)
 	validationsRepo := validator.NewRepo(db)
+	aggregateRepo := reports.NewRepository(logger, db)
```

Comment:
Would this make more sense as `reportsRepo := ...`?

Reply:
Oh yeah, `aggregate` was something I used before refactoring, thanks for the catch.